src/test/ledger/Invariants_test.cpp

```diff
@@ -162,6 +176,164 @@ class Invariants_test : public beast::unit_test::suite
             STTx{ttACCOUNT_DELETE, [](STObject& tx) {}});
     }
 
+    void
+    testAccountRootsDeletedClean()
+    {
+        using namespace test::jtx;
+        testcase << "account root deletion left artifact";
+
+        for (auto const& keyletInfo : directAccountKeylets)
+        {
+            // TODO: Use structured binding once LLVM issue
+            // https://github.com/llvm/llvm-project/issues/48582
+            // is fixed.
```

Comment:
Looks like this issue has been fixed: https://github.com/llvm/llvm-project/issues/48582

Reply:
Nice. Looks like it'll be in clang 16. Think I should update the comment?

Reply:
Updated in 527a964